hw3/01-characters.js

```diff
@@ -0,0 +1,69 @@
+async function addCharacters(charactersInformation) {
+  const dataContainerElement = document.querySelector('.dataContainer');
+  const row = document.createElement('div');
+  row.className = 'row';
+
+  charactersInformation.forEach((character) => {
+    // Creating the main div with Bootstrap column classes
+    const parentDivElement = document.createElement('div');
+    parentDivElement.className = 'col-6 col-md-4 col-lg-3 mt-2 mx-0';
+
+    // div element for an image
+    const imageDivEement = document.createElement('div');
+    imageDivEement.className = 'pt-2 justify-content-center mx-auto imageDivEementClass';
+    imageDivEement.style = 'text-align: center; width: 190px; height: 340px; margin-top: 20px;  padding: 0px;';
+
+    // Add hover background color change
+    imageDivEement.addEventListener('mouseover', () => {
+      imageDivEement.style.backgroundColor = 'var(--blue)';
```

Comment:
Ariel pointed out to me in mine that event listeners aren't needed here. You can optionally use css pseudo-selectors instead.